scripts/scripts.js

```diff
@@ -316,3 +316,11 @@ export const convertExcelDate = (excelDate) => {
   const parsed = excelTimestampAsUnixTimestamp + delta;
   return Number.isNaN(parsed) ? null : new Date(parsed);
 };
+
+/**
+ * Prepends an image path for images depending on window location
+ */
+export const prependImage = (imagePath) => {
+  const host = window.location.origin.startsWith('http://localhost') ? 'https://main--helix-sportsmagazine--headwirecom.hlx.live' : window.location.origin
```

Comment:
@DriftingSands  You can replace `.hlx.live` with `.hlx.page` so that localhost uses the preview instead of the production env. 

.page => preview
.live => production